docs/cli.md

```diff
@@ -0,0 +1,714 @@
+  schedule list  Show schedule list
+  schedule show  Show a schedule
+  subscription create  Create subscription
+  subscription delete  Delete subscription
+  subscription list  Show subscription list
+  subscription show  Show a subscription
+```
+
+#### Parameters
+
+| Parameter                             | Description |
+| ------------------------------------- | -- |
+| --version                             | Show program's version number and exit |
+| -v, --verbose, --debug                | Increase verbosity of output and show tracebacks on errors. You can repeat this option. |
+| -q, -quiet                            | Suppress output except warnings and errors. |
+| --log-file LOG_FILE                   | Specify a file to log output. Disabled by default. |
+| -h, --help                            | Show help message and exit. |
+
+# Meteoroid CLI Commands
+
+### endpoint create
+
+Create an endpoint.
+
+```bash
+meteoroid endpoint create                 
+usage: meteoroid endpoint create [-h] [--fiwareservice FIWARESERVICE] [--fiwareservicepath FIWARESERVICEPATH]
+                                 name path method function_id
+```
+
+#### Parameters
+
+| Parameter                             | Description |
+| ------------------------------------- | -- |
+| --fiwareservice FIWARESERVICE         | FIWARE Service. |
+| --fiwareservicepath FIWARESERVICEPATH | FIWARE Service Path. |
+| name                                  | Name of endpoint to be created. |
+| path                                  | Path for calling Endpoint. Path must begin with '/'. |
+| method                                | HTTP Method for calling the endpoint. |
+| function_id                           | Function ID associated with Endpoint. |
+
+#### Usage example
+
+```bash
+meteoroid endpoint create endpoint1 /function1 post 4
+{'id': 1, 'url': 'http://192.168.0.1:9090/api/23bc46b1-71f6-4ed5-8c54-816aa4f8c502/endpoint1/function1', 'fiware_service': '', 'fiware_service_path': '/', 'name': 'endpoint1', 'path': '/function1', 'method': 'post', 'function': 1}
+```
+
+### endpoint delete
+
+Delete an endpoint.
+
+```bash
+meteoroid endpoint delete
+usage: meteoroid endpoint delete [-h] [--fiwareservice FIWARESERVICE] [--fiwareservicepath FIWARESERVICEPATH] id
+```
+
+#### Parameters
+
+| Parameter                             | Description |
+| ------------------------------------- | -- |
+| --fiwareservice FIWARESERVICE         | FIWARE Service. |
+| --fiwareservicepath FIWARESERVICEPATH | FIWARE Service Path. |
+| id                                    | Endpoint ID. |
+
+#### Usage example
+
+```bash
+meteoroid endpoint delete 1
+```
+
+### endpoint list
+
+List endpoints.
+
+```bash
+meteoroid endpoint list
+```
+
+#### Usage example
+
+```bash
+meteoroid endpoint list
++----+----------------------------------------------------------------------------------+----------------+---------------------+-----------+--------+--------+----------+
```

Comment:
Bash syntax highlighting has been applied to the results.
I have an idea that is to split the meteoroid-cli command and the result.
What do you think?

Like this.
```bash
meteoroid endpoint list
```

```
+----+----------------------------------------------------------------------------------+----------------+---------------------+-----------+--------+--------+----------+
....
```

Reply:
It is certainly a bad highlight.
I'll fix.

Reply:
I removed Bash syntax highlighting.